admin/SWC_cover_letter.txt

```diff
@@ -0,0 +1,22 @@
+Dear SCF Mentoring Subcommittee,
+
+We are writing to express our interested to attend the two-day training course in December 2015 as one group composed of:
+
+Madeleine Bonsma (@mbonsma on GitHub),
+Lindsay Coome (@lcoome on GitHub),
+Lina Tran (@linamnt on Github),
+Sarah Meister (@thesarahmeister on GitHub),
+Elliott Sales de Andrade (@QuLogic on GitHub),
+Ricardo Harripaul (@ricardoharripaul on GitHub),
+Derek Howard (@derekhoward on Github), and
+Dawn Walker (@dcwalk on Github).
+
```

Comment:
@edickie, great! I'll put you down on the cover letter to instruct for R because we're short on people in that area right now, if that's alright. We might switch things around when the sessions actually come.